containers/Dockerfile.base-spack

```diff
@@ -145,14 +147,14 @@ FROM base-spack as dev
 ARG MPI_NAME=openmpi
 ARG MPI_VERSION=4.0.2
 
-RUN spack load ${COMPILER_PACKAGE}@${COMPILER_VERSION} \
+RUN spack load ${COMPILER_PKG_NAME}@${COMPILER_PKG_VERSION} \
  && spack compiler find
 
 RUN spack install \
-    cmake%${COMPILER_NAME}@${COMPILER_VERSION} arch=${SPACK_ENV_ARCH} \
-    gmake%${COMPILER_NAME}@${COMPILER_VERSION} arch=${SPACK_ENV_ARCH} \
+    cmake%${COMPILER_NAME}@${COMPILER_VERSION} arch=${ENV_SPACK_ARCH} \
+    gmake%${COMPILER_NAME}@${COMPILER_VERSION} arch=${ENV_SPACK_ARCH} \
```

Comment:
I asssume the `${ENV_COMPILER_NAME}` and `${ENV_COMPILER_VERSION}` isn't required here because it wasn't before, but just thought I'd point it out in case it should be.

Reply:
I'll add the `ENV_` prefix. Thanks!